lib/nuget-parser/parsers/csproj-parser.ts

```diff
@@ -24,62 +24,64 @@ function findFile(rootDir, filter) {
   return;
 }
 
-export async function getTargetFrameworksFromProjFile(
+export function getTargetFrameworksFromProjFile(
```

Comment:
No async stuff was done here, so simplified the logic to stop all the `Promise` callback juggling and just return an array always. Either populated or empty.